burner-wallet-plugin/tokenbridge-bw-exchange/src/burner-wallet/assets/BridgeableERC20Asset.ts

```diff
@@ -1,42 +1,55 @@
 import { ERC20Asset } from '@burner-wallet/assets'
-import { MEDIATOR_ABI, constants } from '../../utils'
-import { toBN } from 'web3-utils'
+import { AssetConstructor } from '@burner-wallet/assets/Asset'
+import { MEDIATOR_ABI, constants, isBridgeContract } from '../../utils'
+import { toBN, soliditySha3 } from 'web3-utils'
 
-interface BridgeableERC20Constructor {
+interface BridgeableERC20Constructor extends AssetConstructor {
   abi?: object
   address: string
-  id: string
-  name: string
-  network: string
-  bridgeAddress: string
+  bridgeModes?: string[]
```

Comment:
`bridgesModes` is defined as an optional parameter but it receives a default value in case it is not provided in the constructor. Does it make sense to change the type to `bridgeModes: string[]`?

Reply:
I think it makes sense. Thank you.